businesscentral-monitoring/image.yaml

```diff
@@ -156,6 +156,7 @@ packages:
 osbs:
       configuration:
           container_file: container.yaml
+      extra_dir: custom-files
```

Comment:
Can we call this something more specific then "custom-files"? That is vague and doesn't really say what the purpose is. I think even the name "osbs_extra" (from the cekit docs) would be fine, as the name implies what it's for.

Reply:
dir can called "cvp" as the content is required for cvp, but in future there can be more files that are not related to cvp so "osbs-extra" will be probably better option

Reply:
Agreed that it's a better idea to make it a name for any extra files that osbs might need - whether it be now or in the future.

Reply:
@errantepiphany dir name updated

Reply:
thanks; looks good